geopmdpy/geopmdpy/session.py

```diff
@@ -614,8 +617,8 @@ def get_parser():
                         help='Print version and exit.')
     parser.add_argument('-t', '--time', dest='time', type=float, default=0.0,
                         help='Total run time of the session to be opened in seconds.')
-    parser.add_argument('-p', '--period', dest='period', type=float, default=0.0,
-                        help='When used with a read mode session reads all values out periodically with the specified period in seconds.')
+    parser.add_argument('-p', '--period', dest='period', type=float, default=0.1,
```

Comment:
Does this break one-shot mode due to the check on line 301?

Reply:
Yes...yes it does